### PR TITLE
FM2-510: FHIR IG Group Resource Profile, Example

### PR DESCRIPTION
**Pull request by abertnamanya**
Issue Ticket: https://issues.openmrs.org/browse/FM2-510

Screenshot from 2022-09-09 18-53-20

Screenshot from 2022-09-09 18-53-06

### CODE REVIEW

**Review by pmanko — APPROVED** [review]
looks great!

File: input/fsh/omrs-group.fsh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/** OMRS Group resource  profile*/
+
+Profile: OMRSGroup
+Parent: Group
+Id: omrs-group
+Title: "OMRS Group"
+Description: "A Group Resource is described as Cohort in Fhir2 OMRS"
+* identifier 1..1
```
Comment (pmanko) [inline input/fsh/omrs-group.fsh:8]: I don't think the fhir2 module supports `identifier`, `managingEntity`, or `characteristic`, but we do have a description extension.

**Review by pmanko — CHANGES_REQUESTED** [review]
Let's double-check the fields to only represent what's implemented in the fhir2 module.

File: input/fsh/omrs-group.fsh (hunk 1 above)
Comment (abertnamanya) [reply input/fsh/omrs-group.fsh:8]: Thank you @pmanko True I have checked identifier and characteristic are not supported by fhir2 module but for managingEntity was referenced in one of the test cases https://github.com/openmrs/openmrs-module-fhir2/blob/d0b34f74c243b32669d86c9369f7b226a0ada2f9/api-2.1/src/test/java/org/openmrs/module/fhir2/api/translators/impl/GroupTranslatorImpl_2_1Test.java#L223

Comment (abertnamanya) [reply input/fsh/omrs-group.fsh:8]: I think also  its being consumed by the baseGroupTranslastor https://github.com/openmrs/openmrs-module-fhir2/blob/d0b34f74c243b32669d86c9369f7b226a0ada2f9/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseGroupTranslator.java#L73

**Review by pmanko — APPROVED** [review]

**Pull request merged**